students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md

```diff
@@ -33,15 +45,58 @@ Information Systems Intern
 *Responsibilities:*<br>
 
 * Providing advice regarding integration of Microsoft Office into business processes.
+* Studying usage issues pertaining to Microsoft Excel in the financial department, and proposing solutions to improve the workflow.
+
+## Project Experiences
+
+### Contributions to OSS Projects
+
+External Projects:
+* [GitHub's Electron](https://github.com/electron/electron) (Contributor)
+    * [Fix webpreferences not accepting numeric options #8542](https://github.com/electron/electron/pull/8542)
+    * [Support message, nameFieldLabel and showsTagField for macOS dialog #8556](https://github.com/electron/electron/pull/8556)
+
+Internal Projects (for CS3281&2: Thematic Systems Project I/II):
```

Comment:
Just list the project in the order of importance. No need to mention external/internal or module codes.